src/views/router_views/psc_type/psc_type.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block main_content %}
+
+{% set pscTypeTitle = i18n.psc_type_title %}
+{% set pscTypeIndividual = i18n.psc_type_individual %}
+{% set pscTypeRle = i18n.psc_type_rle %}
+{% set pscTypeHint = i18n.psc_type_hint | safe %}
+{% set detailsTitleText = i18n.psc_type_details_title %}
+{% set detailsText = i18n.psc_type_details_text %}
+{% set continueText = i18n.global_confirm_continue_button %}
```

Comment:
I like the idea of putting our variables at the top of the page. I think that makes the code more readable.
However, you don't need to create variables for most of these translations. The i18n.xxx will work just fine.